util/cmake/SeqAnBuildSystem.cmake

```diff
@@ -437,12 +437,20 @@ macro (seqan_configure_cpack_app APP_NAME APP_DIR)
   # The following include automates the MS Redistributable installer.
   include (InstallRequiredSystemLibraries)
 
+  SET(CPACK_GENERATOR "ZIP")
```

Comment:
We only need ZIP for mac and windows, not linux and bsd (they have the tarballs).